DBConnection/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Logging;
+
+namespace DBConnection
+{
+    public class Program
+    { 
+        
+        public static void Main(string[] args)
+        {
+            Log Log = new Log();
+            Log.InitializeLog();
+            try
+            {
+              Log.Info("Log Initialisiert!");
+              Log.Info("Versuche nun mit der Datenbank zu Verbinden..");
+              Connection oConnection = new Connection();
+              oConnection.DBConnect();
+              if (oConnection.DBConnect() == true)
```

Comment:
```suggestion
              if (oConnection.DBConnect())
```
Ein Vergleich mit myBoolean == true ist redundant, weil der Boolean an sich schon den Wahrheitswert hat 👍